.changeset/perfect-wasps-grow.md

```diff
@@ -0,0 +1,20 @@
+---
+'astro': patch
```

Comment:
It should be a minor because we expose a new API to users.

Reply:
Oh, that's good to know! Sounds like this should coincide with the stable release. I'll remove the `deprecated` flags and just remove `z` from the types.